waku-wacky-whisper-spec.md

```diff
@@ -0,0 +1,46 @@
+# Waku Wacky Whisper Specification
+
+> Version: 0.1 (Draft)
+
+> Authors: Infura "The Devil" Osaka <noreply@status.im>
+
+Basically Infura for chat. Addresses scalability issues of Whisper by sacrificing metadata protection. Specifically for resource restricted devices like mobilephones with limited data plan.
+
+インフラ (Infura) means infrastructure in Japanese, and 枠 (Waku) is a closely related word that means 'frame' or 'slide'.
+
+## Rationale
+
+Many users put a premium on bandwidth usage over metadata protection. Implementing Waku mode gives users a more performant experience, at the expense of metadata protection and requiring stronger connectivity.
+
+For more background, please see Whisper theoretical model [report](https://htmlpreview.github.io/?https://github.com/vacp2p/research/blob/master/whisper_scalability/report.html) with associated [source code](https://github.com/vacp2p/research/tree/master/whisper_scalability).
+
+## Roles and definitions
+
+1. Waku-chan
```

Comment:
I would prefer to use more explicit client and server terms, even if it loses the "joke"

Reply:
yes, waku client  / waku node is better, otherwise we need to have context on which one is which

Reply:
Sure thing, agee